app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
+use Transoft\Blog\Block\Bloglist;
+?>
+<?php /** @var Bloglist $block*/
+?>
+<?php
+$bloglist = $block->getData('bloglist');
+$data = $bloglist->getPostCollection();
+?>
+
+<?php if ($bloglist->isTypesMatching()): ?>
+    <div>
+        <h3><?= $block->escapeHtml(__('Latest 5 blogs:')) ?></h3>
+        <ul>
+            <?php foreach ($data as $value):
+                $theme = $value->getTheme();
+                $url = $block->escapeUrl($bloglist->getUrlById($value->getBlogId()));
+                ?>
+                <li><a href="<?= $block->escapeUrl($url) ?>"><?= $block->escapeUrl($theme) ?></a></li>
```

Comment:
Why **escapeUrl($theme)**? Please refer to [XSS protection guide](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/xss-protection.html) once again

Reply:
fixed